spring-test/src/test/java/org/springframework/test/util/ReflectionTestUtilsTests.java

```diff
@@ -46,7 +46,7 @@
  */
 class ReflectionTestUtilsTests {
 
-	private static final Float PI = Float.valueOf((float) 22 / 7);
+	private static final Float PI = (float) 22 / 7;
```

Comment:
As a side note, I wouldn't refer to these changes as "Removed unnecessary boxing".

The boxing is necessary. It's just that the compiler will perform auto-boxing for you if you don't do it yourself.

So I'd probably use "Rely on auto-boxing where appropriate" as the commit message title.

Reply:
Changed commit message [e41e9e1](https://github.com/spring-projects/spring-framework/pull/29413/commits/e41e9e16f30a2dec91cd5cd94d78197d1bc187e3)